em-app/src/csr.rs

```diff
@@ -90,24 +91,24 @@ impl ExternalKey for Pk {
             PkType::Eckey | PkType::Ecdsa => Ok(SignatureAlgorithm::EcdsaX962),
             _ => Err(Error::ExternalKeyString(format!("Invalid key type: {:?}", self.pk_type()))),
         }?;
-        
+
         Ok((sig, sigalg))
     }
 }
 
-pub fn get_csr_common_name(signer: &mut dyn CsrSigner,
-                           common_name: &str,
-                           attributes: Vec<(ObjectIdentifier, Vec<Vec<u8>>)>,
-                           extensions: &Option<Vec<(ObjectIdentifier, bool, Vec<u8>)>>
-) -> Result<String> {
-    let subject = vec![(pkix::oid::commonName.clone(), TaggedDerValue::from_tag_and_bytes(TAG_UTF8STRING, common_name.as_bytes().to_vec()))].into();
-    get_csr(signer, &subject, attributes, extensions)
+pub(crate) fn common_name_to_subject(common_name: &str) -> Name {
```

Comment:
I think this function can be moved to lib.rs

Reply:
Moved to `lib.rs`